src/tests/ftest/cart/no_pmix/multictx_one_node.yaml

```diff
@@ -13,7 +13,7 @@ env_CRT_PHY_ADDR_STR: !mux
   sm:
     CRT_PHY_ADDR_STR: "sm"
   ofi_tcp:
-    CRT_PHY_ADDR_STR: "ofi+tcp;ofi_rxm"
+    CRT_PHY_ADDR_STR: "ofi+tcp"
```

Comment:
i would keep both ofi_tcp and ofi_tcp_rxm: variants in this test as we already test sm too here